docs/transparent-validation/lab-assembly-guides/ms_z690-lab-setup.md

```diff
@@ -0,0 +1,114 @@
+# MSI PRO Z690-A WIFI DDR4 Lab Setup
```

Comment:
Just looking at the name of the document: what is ms_z690? Please, name the document in accordance with the full name of the platform or platforms family - otherwise, we are on a straight path to getting lost in our own documentation

Reply:
Changed